requirements.txt

```diff
@@ -146,6 +146,7 @@ Send2Trash==1.8.0
 shortuuid==1.0.8
 shtab==1.5.3
 six==1.16.0
+sklearn==0.0
```

Comment:
I suppose this was added by pip freeze? If so, I suppose that's OK, although I just read about this [here](https://towardsdatascience.com/scikit-learn-vs-sklearn-6944b9dc1736):
"it is recommended to install scikit-learn through pip using the scikit-learn identifier."

Reply:
It was added by pip freeze but if I remember correctly it was because I ran `pip install sklearn` instead of scikit-learn. I'll remove this since it's redundant